code/__DEFINES/jobs.dm

```diff
@@ -101,3 +102,4 @@
 #define JOB_DISPLAY_ORDER_BLUESHIELD 37
 #define JOB_DISPLAY_ORDER_BO 38
 #define JOB_DISPLAY_ORDER_PRISONER 39
+#define JOB_DISPLAY_ORDER_STOWAWAY 40
```

Comment:
This allows to display stowaways at the bottom of the occupation list. Right after the prisoners.